docs/oss/security/apache/activemq.md

```diff
@@ -94,18 +120,18 @@ Here is how the rating was calculated:
       Value:........2.41  out of 10.00
       Confidence:...10.00 out of 10.00
       Based on:...2 features:
-          Number of stars for a GitHub repository:......1729
+          Number of stars for a GitHub repository:......1734
           Number of watchers for a GitHub repository:...203
 
       Sub-score:....Vulnerability lifetime
       Importance:...Low (weight 0.23 out of 1.00)
       Value:........0.00  out of 10.00
       Confidence:...10.00 out of 10.00
       Based on:...2 features:
-          Info about vulnerabilities:...27 vulnerabilities
+          Info about vulnerabilities:...32 vulnerabilities
```

Comment:
Could this be a `FALSE POSITIVE` case

[link](https://nvd.nist.gov/vuln/search/results?form_type=Advanced&results_type=overview&search_type=all&cpe_vendor=cpe%3A%2F%3Aapache&cpe_product=cpe%3A%2F%3A%3Aactivemq) returns only 29 results.

Reply:
I guess we are not there yet with the accuracy.

Reply:
Note that the report was created when https://github.com/SAP/fosstars-rating-core/pull/180 was not merged.

Reply:
Right now it gives `30 vulnerabilities`.

Reply:
Here I see 28

https://www.cvedetails.com/product/19047/Apache-Activemq.html?vendor_id=45

Reply:
You know this can be confirmed with #189. As the REST API returns the total number of Vulnerabilities associated to a CPE. Maybe we use that to make this process more intelligent.

Reply:
Sure, we should definitely look into it.